src/Ledger/Conway/Specification/Chain/Properties/GovDepsMatch.lagda.md

```diff
@@ -106,14 +107,17 @@ In other terms,
 *Proof*.
 
 ```agda
-  CHAIN-govDepsMatch rrm rss (CHAIN (x , NEWEPOCH-New (_ , eps₁→eps₂) , ledgers)) =
+  CHAIN-govDepsMatch rrm rss
+      (CHAIN (x , TICK ((NEWEPOCH-New (_ , eps₁→eps₂)) , _) , ledgers)) =
```

Comment:
Why are arguments to data constructors passed in tuples here? The data constructors are curried at their definitions 🤔

Reply:
They are actually defined with tuples via bullet points.